src/Strategy/Gomodules.hs

```diff
@@ -75,6 +77,8 @@ getDeps project = do
       , dependencyManifestFiles = [gomodulesGomod project]
       }
   where
+    -- `go list -json -deps all` marks std lib deps with a boolean, so Strategy.Go.GoListPackages does this filtering itself.
+    -- I think this can be removed when it becomes the default.
```

Comment:
```suggestion
    -- I think this can be removed when `go list -json -deps all` becomes the default.
```

Be explicit instead of `it`.

Also just out of curiosity, the reason this can be removed is because we are explicity filtering the `stldlib` deps during graph creation in the new method right?